pyccel/codegen/printing/ccode.py

```diff
@@ -385,6 +386,22 @@ def get_declare_type(self, expr):
         else:
             return '{0} '.format(dtype)
 
+    def _print_FuncAddressDeclare(self, expr):
+        if len(expr.results) == 1:
+            ret_type = self.get_declare_type(expr.results[0])
+        elif len(expr.results) > 1:
+            msg = 'Multiple output arguments is not yet supported in c'
+            errors.report(msg+'\n'+PYCCEL_RESTRICTION_TODO, symbol=expr,
+                severity='fatal')
+        else:
+            ret_type = self._print(datatype('void')) + ' '
+        name = expr.name
+        if not expr.arguments:
+            arg_code = 'void'
+        else:
+            arg_code = ', '.join('{}'.format(self._print_FuncAddressDeclare(i)) if isinstance(i, FunctionAddress) else '{0}{1}'.format(self.get_declare_type(i), i) for i in expr.arguments)
```

Comment:
This line doesn't fit on my screen on github. It needs to be split into a multi-line object